website/markdown/docs/commands/dependencies.mdx

```diff
@@ -0,0 +1,211 @@
+
+<EnumTable
+  cases={[
+    {
+      case:
+        '.carthage(origin: CarthageOrigin, requirement: CarthageRequirement, platforms: Set<Platform>)',
+      description:
+        'A description of dependency that can by installed using Carthage',
+    },
+  ]}
+/>
+
+### CarthageOrigin
+
+Origin of the Carthage dependency.
+
+<EnumTable
+  cases={[
+    {
+      case: '.github(path: String)',
+      description:
+        'Mimics `github` keyword from `Cartfile`. GitHub repositories (both GitHub.com and GitHub Enterprise).',
+    },
+    {
+      case: '.git(path: String)',
+      description:
+        'Mimics `git` keyword from `Cartfile`. Other Git repositories.',
+    },
+    {
+      case: '.binary(path: String)',
+      description:
+        'Mimics `binary` keyword from `Cartfile`. Dependencies that are only available as compiled binary `.frameworks`.',
+    },
+  ]}
+/>
+
+### CarthageRequirement
+
+Requirement for the Carthage dependency.
+
+<EnumTable
+  cases={[
+    {
+      case: '.exact(Version)',
+      description: 'Mimics `== 1.0` from `Cartfile`.',
+    },
+    {
+      case: '.upToNext(Version)',
+      description: 'Mimics `~> 1.0` from `Cartfile`.',
+    },
+    {
+      case: '.atLeast(Version)',
+      description: 'Mimics `>= 1.0` from `Cartfile`.',
+    },
+    {
+      case: '.branch(String)',
+      description: 'Mimics `"branch"` from `Cartfile`.',
+    },
+    {
+      case: '.revision(String)',
+      description: 'Mimics `"revision"` from `Cartfile`.',
+    },
+  ]}
+/>
+
+## Commands
+
+The folder structure below shows how Tuist organize dependenies:
+
+```bash
+Tuist
+	|- Dependencies.swift # Manifest
+  |- Dependencies
+    |- graph.json # comming soon
+    |- Lockfiles
+      |- Carthage.resolved
+      |- Podfile.lock # comming soon
+      |- Package.resolved # comming soon
+    |- Carthage # stores content of `Carthage/Build` generated by `carthage`
+      |- iOS
+         |- Alamofire.framework
+      |- tvOS
+         |- Alamofire.framework
+    |- Cocoapods # comming soon
+      |- RxSwift
+    |- SwftPackageManager # comming soon
+      |- Moya
+```
+
+### Fetching
+
+```bash
+tuist dependencies fetch
+```
+
+<ArgumentsTable
+  args={[
+    {
+      long: '`--path`',
+      short: '`-p`',
+      description:
+        'The path to the directory that contains the workspace or project whose dependencies will be fetched.',
+      default: '',
+      optional: true,
+    },
+  ]}
+/>
+
+### Updating
+
+```bash
+tuist dependencies update
+```
+
+<ArgumentsTable
+  args={[
+    {
+      long: '`--path`',
+      short: '`-p`',
+      description:
+        'The path to the directory that contains the workspace or project whose dependencies will be updated.',
+      default: '',
+      optional: true,
+    },
+  ]}
+/>
+
+## Link dependencies
+
+You can link third-party dependencies like local ones.
+
+```swift
+let target = Target(dependencies: [
+  .framework("Tuist/Dependencies/Carthage/iOS/Alamofire.framework"),
```

Comment:
not related to this PR specifically: should not we provide a simplified API for this? Or is that planned? But as we own the Carthage integration, it'd be also nice to make the location of the framework inherent without user explicitly specifying it